packages/esm-ugandaemr-app/src/ugandaemr-configuration-overrrides.json

```diff
@@ -2,34 +2,113 @@
   "@openmrs/esm-appointments-app": {
     "Display conditions": {
       "privileges": [
-        ""
+        "Manage Appointments"
       ]
     }
   },
   "@openmrs/esm-patient-chart-app": {
     "Display conditions": {
       "privileges": [
-        ""
+        "View Patient Summary"
       ]
-    },
-    "extensionSlots": {
-      "action-menu-non-chart-items-slot": {
-        "remove": [
-          "patient-list-action-menu"
-        ]
-      },
-      "cacx-visits-slot": {
-        "remove": [
-          "cacx-visits-ext"
-        ]
-      }
     }
   },
   "@openmrs/esm-styleguide": {
     "Brand color #1": " #009384",
     "Brand color #2": " #009384",
     "Brand color #3": " #009384"
   },
+  "@openmrs/esm-patient-biometrics-app": {
+    "Display conditions": {
```

Comment:
These aren't meant to be here but i guess we can maintain this for now for testing purposes as the backend gets resolved.